### PR TITLE
Add android interview questions

### PR DESCRIPTION
**Pull request by akhilesh0707**
Added new android questions on the list.

### CODE REVIEW

**Review by 53jk1 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -127,6 +127,10 @@ Please read the [contribution guidelines](contributing.md) or the [creating a li
 * [Collection of Android and Java questions divided by experience](https://medium.com/@neteinstein/not-another-android-interviews-article-the-questions-3dedafa30bec)
 * [Android Interview Questions & How to Interview Candidates](https://pangara.com/blog/android-interview-questions)
 * [RocketSkill App Android Interview Questions](https://github.com/mindash/android-structured-interview)
+* [Android cheat sheet: Coding program, Data structure, Android and Java interview questions with answers and categorized by topics](https://github.com/anitaa1990/Android-Cheat-sheet)
+* [Android Interview Questions And Answers for experienced and fresher level developer](https://www.andreasschrade.com/2017/02/23/android-interview-questions/)
```
Comment (ryan-blunden) [inline README.md:131]: Please change to "Android Interview Questions And Answers From Beginner To Advanced"

File: README.md [hunk 2]
```diff
@@ -127,6 +127,10 @@ Please read the [contribution guidelines](contributing.md) or the [creating a li
...
+* [Android Interview Questions for Senior Android Developer](https://github.com/mohsenoid/Android-Interview-Questions)
```
Comment (ryan-blunden) [inline README.md:132]: Please change to "Interview Questions for Senior Android Developers"

File: README.md (hunk 1 above)
Comment (akhilesh0707) [reply README.md:131]: Hi @ryan-blunden I tried to edit this PR readme file but get an error so I have raised another MR can you review and approve so I can close this one

File: README.md (hunk 2 above)
Comment (akhilesh0707) [reply README.md:132]: Hi @ryan-blunden I tried to edit this PR readme file but get an error, so I have raised another PR with a fix of review comments, can you review and approve so I can close this one.

**Comment by akhilesh0707** [comment]
new PR: https://github.com/DopplerHQ/awesome-interview-questions/pull/230

**Pull request closed without merging**